src/clj_gcp/storage/core.clj

```diff
@@ -39,11 +40,17 @@
 ;;| GCSStorageClient
 ;;`---------------
 (defn- gcs-get-blob
+  "Returns a Truthy determining if a file exists in the remote location provided"
```

Comment:
This doc doesn't describe what `gcs-get-blob` does, just what we're using it for in this use case. Can we make it just describe getting blobs?